index.html

```diff
@@ -650,14 +669,14 @@ <h3 class="primary-text1 about-me-header">About Myself</h3>
             </div>
             <ul class="about-me-skill-list">
                 <li class="select">
-                        <span class="select-text">Languages</span>
+                        <span class="select-text select-text-color">Languages</span>
                         <span class="select-svg">
                             <svg width="14" height="8" viewBox="0 0 14 8" fill="none" xmlns="http://www.w3.org/2000/svg">
                                 <path fill-rule="evenodd" clip-rule="evenodd" d="M0.292893 0.292893C0.683417 -0.0976311 1.31658 -0.0976311 1.70711 0.292893L7 5.58579L12.2929 0.292893C12.6834 -0.0976311 13.3166 -0.0976311 13.7071 0.292893C14.0976 0.683417 14.0976 1.31658 13.7071 1.70711L7.70711 7.70711C7.31658 8.09763 6.68342 8.09763 6.29289 7.70711L0.292893 1.70711C-0.0976311 1.31658 -0.0976311 0.683417 0.292893 0.292893Z" fill="#5E6C84"/>
                             </svg>
                         </span>
```

Comment:
Please add the grey box around the languages to fit with the Figma design and avoid overlaps. 😄

Figma design:
![image](https://user-images.githubusercontent.com/42799579/192801622-77b46e01-59dc-4afd-9fba-7f41cb94f4c5.png)

Your website:
![image](https://user-images.githubusercontent.com/42799579/192801516-88fe1ac7-6389-4eec-82c5-d82e52b4c112.png)

Reply:
Fixed.